presidio_evaluator/evaluation/evaluator.py

```diff
@@ -5,23 +5,29 @@
 import string
```

Comment:
Changes here:
- added stop_words to skip cases where there are errors but it's likely not PII
- added generic_entities which would not be considered as a mismatch if detected. For example, if a model detects a SSN as an `ID` entity, this wouldn't be considered an error as many models just output `ID` or `PII` for numbers that are likely PII.
- Updated the evaluation loop to consider generic_entities and stop_words